### ISSUE TITLE
quote delimiters get changed

### ISSUE THREAD

**Issue by rowlesmr** [sourcery-ai/sourcery#221]
### Issue description or question

A suggestion to alter some string manipulation changes double quotes to single quotes

![image](https://user-images.githubusercontent.com/14921460/162365600-0a5561ed-9f60-41f1-8980-772574475404.png)

before
```
    while len(s) < d:
        if pad == "post":
            s = s + " "
        elif pad == "pre":
            s = " " + s
        else:
            return s
    return s
```

after
```
    while len(s) < d:
        if pad == "post":
            s = f'{s} '
        elif pad == "pre":
            s = f" {s}"
        else:
            return s
    return s
```

### Sourcery Version

0.11.2

### Code editor or IDE name and version

Pycharm v2021.3.3

### OS name and version

Windows 10

**Comment by ruancomelli** [comment]
Hi @rowlesmr! Thanks for flagging this issue. I was able to reproduce it and am already working on a fix.

**Comment by ruancomelli** [comment]
A fix for this issue will be available in the next Sourcery release, thanks again for bringing this up!

**Comment by Hellebore** [comment]
Fixed in 0.11.3
